atom/common/api/atom_api_clipboard.cc

```diff
@@ -50,9 +51,49 @@ std::string Read(const std::string& format_string,
   return data;
 }
 
+void Write(const std::vector<mate::Dictionary>& data,
+           mate::Arguments* args) {
+  ui::ScopedClipboardWriter writer(GetClipboardType(args));
+  for (uint32_t i = 0; i < data.size(); ++i) {
+    base::string16 text;
+    gfx::Image image;
+    std::string format;
+    ui::Clipboard::FormatType type;
+    if (data[i].Get("format", &format))
```

Comment:
Since the ScopedClipboardWriter doesn't allow writing arbitrary types, I don't think we should allow `format` option here, it mistakingly assumes users can write arbitrary date.

Reply:
I want multiple image. how!